aj.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import "github.com/gin-gonic/gin"
+
+func main() {
+      router := gin.Default()
+      router.GET("/hello", func(c *gin.Context) {
+c.JSON(200, gin.H{
+"message":"Hello World!",
+})
+})
+      router.GET("/greet/:name", func(c *gin.Context) {
+	name := c.Param("name")

```

Comment:
Make `"name"` a global constant in this file.

Reply:
ok

Reply:
should i make it an global vaariable or global constant?